framework/wazuh/core/indexer/events.py

```diff
@@ -24,7 +24,7 @@ async def create(self, events: List[StatefulEvent]) -> dict:
         # TODO(#24713): Implement server to indexer events batching
         response = {}
         for i, event in enumerate(events.events):
-            resp = await self._client.create(index=event.get_index_name(), body=asdict(event))
+            resp = await self._client.index(index=event.get_index_name(), body=asdict(event))
```

Comment:
This is a typo I noticed during the tests, I figured we could include it directly in this PR